db/migrate/20220608204929_create_admins.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAdmins < ActiveRecord::Migration[7.0]
+  def change
+    create_table :admins do |t|
+      t.string :name
+      t.string :password
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
Are you planning to use a personal session setup or are you going to use a user setup gem like Devise or Clearence?

Reply:
Authentication from Scratch no gem